src/main/java/calculator/util/delimiter/InvalidCustomDelimiter.java

```diff
@@ -0,0 +1,30 @@
+package calculator.util.delimiter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum InvalidCustomDelimiter {
+
+    DOUBLE_SLASH("//"),
+    NEWLINE("\\n"),
+    BLANK(" "),
+    NULL(""),
+    NUMBER("[0-9]");
```

Comment:
```suggestion
public enum InvalidCustomDelimiter {

    DOUBLE_SLASH("//"),
    NEWLINE("\\n"),
    BLANK(" "),
    EMPTY(""),
    NUMBER("[0-9]");
```
Java의 null과 헷갈리기 쉬울 것 같아요. EMPTY와 같이 빈 문자열을 더욱 잘 표현할 수 있는 값을 사용하는 것은 어떨까요?

Reply:
오 EMPTY가 더 직관적인 것 같아요! 수정하겠습니다 :)